libraries/chain/dbs_account.cpp

```diff
@@ -154,6 +156,8 @@ void dbs_account::create_account_with_delegation(const account_name_type& new_ac
 
     if (fee.amount > 0)
         db_impl().create_vesting(new_account, fee);
+
+    return new_account;
```

Comment:
where are you going to use this?

Reply:
maybe in unit tests?)

Reply:
This is the logic of Create pattern. It is used in chainbase::database::create